src/components/LoanCards/MatchingText.vue

```diff
@@ -0,0 +1,13 @@
+<template functional>
+	<div class="matching-text-wrapper" v-if="props.matchingText">
+		<span>2x matching by {{ props.matchingText }}</span>
+	</div>
+</template>
+
+<style lang="scss">
```

Comment:
It may not be necessary, but I've been marking all my component <style> sections as "scoped". Wonder if we should make this a general practice?

Reply:
Just looked that up @jedlin-kiva, thanks for catching that miss of mine. Seems to make sense to me to add the <style scoped> if it's a functional component like this one.

Reply:
"scoped" can have some unexpected behavior, and in general I think it's only useful for pages or components that will only be on the page once.